Create welcome.rst

hillaryfraley opened the pull request:
A rough draft to get us started.

I made my best guesses on link format and deadlines/tickets-on-sale dates based on the Portland content.

hillaryfraley committed:
Create welcome.rst

A rough draft to get us started.

I made my best guesses on link format and deadlines/tickets-on-sale dates based on the Portland content.

plaindocs commented:
Hey @hillaryfraley welcome :-)

You can preview that here https://deploy-preview-485--dentist-canary-64055.netlify.com/conf/cincinnati/2018/news/welcome/ (click 'Show all checks', Netlify 'Details', in the green box :arrow_down: : )

Looks like a great start

ericholscher commented on docs/conf/cincinnati/2018/news/welcome.rst:
```diff
@@ -0,0 +1,63 @@
+
+Welcome to the official launch of `Write the Docs Cincinnati + Open Help 2018 <http://www.writethedocs.org/conf/cincinnati/2018/>`_! We are excited to announce the first-ever collaboration between Write the Docs and Open Help, coming this August.
+
+Since 2011, the Open Help Conference in Cincinnati featured attendee-led discussions and sprints for a wide range of people working on open-source and community-based projects. This year, Write the Docs and Open Help are coming together to bring 5 days of talks, conversations, and doc sprints a little closer to home for our East Coast and Midwestern documentarians!
```
👍

hillaryfraley replied:
I added a sentence at the end of the second paragraph to address @ericholscher 's "learn, discuss, do" comment:

"Write the Docs Cincinnati + Open Help 2018 creates a time and place for the community of documentarians to learn, discuss, and do, working together to improve the art and science of documentation."

ericholscher commented on docs/conf/cincinnati/2018/news/welcome.rst:
```diff
@@ -0,0 +1,63 @@
+* Days 3-5: Documentation sprints and writing days
+
+Write the Docs Cincinnati + Open Help is scheduled for August 18-22, 2018 at `Union Hall <http://unionhallcincy.com/>`_ in the historic Over the Rhine neighborhood in `Cincinnati, Ohio <https://www.google.com/maps/place/Union+Hall/@39.109885,-84.515631,15z/data=!4m5!3m4!1s0x0:0x3caee5bfc65866b3!8m2!3d39.109885!4d-84.515631>`_.
+
```
Feels like we should probably explain a bit more about the specific events. What is an unconference? What do you need to do to prepare for it? We should be able to spin up the "unconf" page on the website and link to it (similar to http://www.writethedocs.org/conf/portland/2018/unconference/), but it feels like we're doing something "more" with it here. 

@shaunix, do you have existing copy here that we can use here?

thatdocslady replied:
We can link to one of our own Unconf/Writing Day topics (on a related note, we can/should probably have a generic-community version of those that's not linked to a specific conference)

hillaryfraley replied:
Thank you -- I added a little more description for each day. If we have generic pages for each day or conference-specific pages, I can add those links.

ericholscher reviewed (commented):
This is a great announcement! There isn't too much I'd add except for a bit more explanation and selling of the event format ("Learn, Discuss, Do")

plaindocs commented:
The only other comment, is do we want to talk a bit more about the WTD+OH collaboration? Or just leave as is?

thatdocslady commented on docs/conf/cincinnati/2018/news/welcome.rst:
```diff
@@ -0,0 +1,63 @@
+Announcing Write the Docs Cincinnati + Open Help Website, Call for Organizers, and CFP
+======================================================================================
+
+Welcome to the official launch of `Write the Docs Cincinnati + Open Help 2018 <http://www.writethedocs.org/conf/cincinnati/2018/>`_! We are excited to announce the first-ever collaboration between Write the Docs and Open Help, coming this August.
```
Maybe linkify the WTD and OH first instance (URL for parent community websites)

hillaryfraley replied:
Thank you -- I added the links. @shaunix looks like the cert for Open Help is still invalid.

thatdocslady commented on docs/conf/cincinnati/2018/news/welcome.rst:
```diff
@@ -0,0 +1,63 @@
+Call for Organizers and Volunteers
+----------------------------------
+
+We need help to make this event a success! If you are interested in volunteering or helping to organize, please email us at cincinnati@writethedocs.org. It's a great opportunity to get involved with the community and also receive a free ticket.
```
we could link to the roles overview of typical conference responsibilities: http://www.writethedocs.org/conf/portland/2018/team/

ericholscher replied:
http://www.writethedocs.org/organizer-guide/confs/event-roles/ is probably the better link.

hillaryfraley replied:
Thank you, I added the link.

thatdocslady commented on docs/conf/cincinnati/2018/news/welcome.rst:
```diff
@@ -0,0 +1,63 @@
+Stay Updated
+------------
+
+Want to find out what's happening with the conference, or enjoy our monthly global community newsletter?
```
should we set up a separate segment in MailChimp for east coast? or just reuse the NA segment?

thatdocslady reviewed (commented):
Looks great, Hillary! Lots of good info but not overwhelming ;) I second most of the feedback comments and added a few nits/suggestions.

hillaryfraley committed:
Update Cincy welcome.rst

* Added links to WTD and OH sites
* Added brief descriptions for each day of the event
* Added link to event roles

hillaryfraley committed:
Add "learn-discuss-do" sentence

Added in second paragraph: "Write the Docs Cincinnati + Open Help 2018 creates a time and place for the community of documentarians to learn, discuss, and do, working together to improve the art and science of documentation."

hillaryfraley commented:
I just realized that the title cuts off at "Announcing Write the Docs Cincinnati + Open Help Website," on the preview: https://deploy-preview-485--dentist-canary-64055.netlify.com/conf/cincinnati/2018/news/welcome/

hillaryfraley committed:
Fix typo (spring --> sprints)
